index.html

```diff
@@ -269,11 +269,42 @@ <h4>Computation steps</h4>
         <li id="step1">Set the <code>root node</code> to the given element, the <code>current node</code> to the <code>root node</code>, and the <code>total accumulated text</code> to the empty string (""). If the <code>root node</code>'s role prohibits naming, return the empty string ("").</li>
         <li id="step2">Compute the text alternative for the <code>current node</code>:
           <ol>
-            <li id="step2A">If the <code>current node</code> is <a class="termref">hidden</a> <strong>and is not</strong> directly referenced by <code>aria-labelledby</code> or <code>aria-describedby</code>, nor directly referenced by a native host language text alternative [=element=] (e.g. <code>label</code> in HTML) or [=attribute=], return the empty string.
+            <li id="step2A">If the <code>current node</code> is <a class="termref">hidden</a> <strong>and is</strong>:
+            <ul>
+              <li><strong>Not</strong> part of an <code>aria-labelledby</code> or <code>aria-describedby</code> traversal, where the node directy referenced by that relation was hidden.</li>
+              <li><strong>Nor</strong> part of a native host language text alternative <a class="termref">element</a> (e.g. <code>label</code> in HTML) or <a class="termref">attribute</a> traversal, where the root of that traversal was hidden.</li>
```

Comment:
ditto

```diff
- was hidden
+ is hidden
```

Reply:
"where the root of that traversal" may also be ambiguous. 

In this case, is "root" referring to the element labeled by this host language text alternative (e.g. an `<input>` labeled by `<label>` that is somewhere inside the overall computation)? Or is this referring to the root of the entire name computation traversal? I think the former because there may not be a time when a hidden node can be the root of the overall computation.

I labored over a diff for a few minutes but this may be similarly confusing. Sharing anyway as a straw man example. 
```diff
- where the root of that traversal
+ where the element referenced by the host language text alternative
```

Reply:
> is "root" referring to the element labeled by this host language text alternative ...?

Yes, that was my intention with that wording. I find the proposed diff fitting, it reuses the terminology used by the original wording which is more appropriate, although it's confusing in the same sense as the original was.

There is already an open issue about this: https://github.com/w3c/accname/issues/148